test/utils/bytecode.hpp

```diff
@@ -333,6 +338,21 @@ inline bytecode sload(bytecode index)
     return index + OP_SLOAD;
 }
 
+inline bytecode tstore(bytecode index, bytecode value)
+{
+    return value + index + OP_TSTORE;
+}
+
+inline bytecode tstore(bytecode index)
```

Comment:
Why do you need this?

Reply:
I reworked a unit test and this was a leftover.